tensorflow/core/BUILD

```diff
@@ -1128,7 +1129,6 @@ tf_cuda_library(
             "example/**/*.cc",
             "framework/**/*.h",
             "framework/**/*.cc",
-            "public/version.h",
```

Comment:
I half expect removing this line to cause a bunch of build errors for things that don't depend on `version_lib`, but we'll see.